101-vm-with-data-management-gateway/azuredeploy.json

```diff
@@ -0,0 +1,306 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "existingDataFactoryName": {
+      "type":"string",
+      "metadata": {
+        "description": "your existing data factory name"
+      }
+    },
+    "GatewayName": {
+      "type": "string",
+      "metadata": {
+        "description": "Gateway name must be unique in subscription"
+      }
+    },      
+    "Size": {
```

Comment:
Move to variable and pick a size suitable for your workload... the superset of allowed values will cause failures in regions that don't support them.